src/seasonal.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * Returns a holiday color. A background and a foreground color.
+ * @return array of 6 integers 2x rgb if today is a holiday , otherwise an empty array.
+ */
+function getHolidayColor(): array
+{
+    $dateToday = mktime(0,0,0,date("m"),date("d"));
+
+    if( $dateToday - mktime(0,0,0,10,31) == 0 )// halloween

```

Comment:
The if could be simplified a lot like this:
```php
if (date("m-d")=="10-31")
{ //its halloween

}
```
I think it's a lot more readable! (And using the date()-method is something that every PHP-Developer is used to so that translates to code that is understandable by 95% of PHP-Developers.